tools/docker/Dockerfile

```diff
@@ -6,7 +6,7 @@
 FROM stellargroup/build_env:ubuntu
 
 # docker build needs to be run in /usr/local
-ADD . /usr/local
+COPY . /usr/local
```

Comment:
I guess this change is because `ADD` is overkill? Did codacy actually catch this?

Reply:
Yes, Codacy caught this. I have no idea what the difference is ;-)